dashboard/client/src/components/ActorTable.tsx

```diff
@@ -70,16 +72,6 @@ const isActorEnum = (state: unknown): state is ActorEnum => {
   return Object.values(ActorEnum).includes(state as ActorEnum);
 };
 
-// We sort the actorsList so that the "Alive" actors appear at first and "Dead" actors appear in the end.
-export const sortActors = (actorList: ActorDetail[]) => {
-  const sortedActors = [...actorList];
-  return _.sortBy(sortedActors, (actor) => {
-    const actorOrder = isActorEnum(actor.state) ? stateOrder[actor.state] : 0;
-    const actorTime = actor.startTime || 0;
-    return [actorOrder, actorTime];
-  });
-};
```

Comment:
looks like we already have sort by state, but is it not working correctly?

#42666 was reported recently.

Reply:
From my testing sort by state for actors seems to be working correctly, I think https://github.com/ray-project/ray/issues/42666 is only for the clusters table

Reply:
oh whoops. i got that mixed up.